libraries/fof/controller/controller.php

```diff
@@ -3347,56 +3347,15 @@ protected function _csrfProtection()
 				break;
 		}
 
-		$hasToken = false;
-		$session  = JFactory::getSession();
+		$token = new \Joomla\Cms\Session\CsrfToken(JFactory::getSession());
```

Comment:
You only know that if FOF is going to be removed. If you're fixing code you have to assume it's for the library and the library is used across Joomla versions